app/services/dark_sky_service.rb

```diff
@@ -14,7 +14,8 @@ def conn
     end
   end
 
-  def get(latitude, longitude)
+  def get(latitude, longitude = nil)
+    latitude, longitude = latitude.to_a if longitude == nil
```

Comment:
What is going on here?  I noticed the `PoiSerializer` calls this with:

```ruby
  def get_forecast
    _weather_service.get(location)
  end
```

Where location = `poi.center` which is a Coordinate object.

Reply:
This allows for the `.get` to accept either a `Coordinate` object (single argument -> longitude == nil) or accept both latitude and longitude

Reply:
So if you send a `Coordinate` object, it is then the latitude argument.

How do you call `latitude.to_a` on a Coordinate object?

Reply:
I see, there is a Coordinate.to_a method. Makes sense now.